spec/px/service/client/hmac_signing_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Px::Service::Client::HmacSigning do
+  subject {
+    Px::Service::Client::Base.tap do |c|
+      c.include(Px::Service::Client::HmacSigning)
+    end.new
+  }
+
+  describe '#make_request' do
```

Comment:
I'd suggest tests where you have two subclasses and make sure you can set separate config for them and read it successfully.